src/app/zap-templates/templates/app/af-structs.zapt

```diff
@@ -16,8 +16,18 @@
 // Struct for {{label}}
 typedef struct _{{asType label}} {
 {{#zcl_struct_items}}
+{{#if (isCharString type)}}
+{{ident}}const {{asUnderlyingZclType type}} {{asSymbol label}};
```

Comment:
Don't you want `const` for all strings and not only `CharString` ?

Reply:
OctetStrings are represented by chip::Span rather instead of char*, which already uses const char *.

Reply:
OctetStrings are represented by chip::Span rather instead of char*, which already uses const char *.